packages/angular-cli/blueprints/ng2/files/__path__/main.ts

```diff
@@ -7,6 +7,8 @@ import { AppModule } from './app/app.module';
 
 if (environment.production) {
   enableProdMode();
+} else {
+  require('zone.js/dist/long-stack-trace-zone');
```

Comment:
Unfortunately this is not the right way to go, as webpack will ALWAYS include that code in your bundle, even in production.